src/core/mount.c

```diff
@@ -1840,9 +1841,8 @@ static int mount_setup_unit(
         assert(options);
         assert(fstype);
 
-        /* Ignore API mount points. They should never be referenced in
-         * dependencies ever. */
-        if (mount_point_is_api(where) || mount_point_ignore(where))
+        /* Ignore API mount points. They should never be referenced in dependencies ever. */
```

Comment:
"Ignore API mount points and \<something\>." Right now the comment doesn't match the code.